apps/web/src/api/hooks/useUpdateOrganizationBranding.tsx

```diff
@@ -1,12 +1,14 @@
 import type { IResponseError } from '@novu/shared';
 import { useMutation, UseMutationOptions, useQueryClient } from '@tanstack/react-query';
-
 import { errorMessage } from '../../utils/notifications';
 import { updateBrandingSettings } from '../organization';
 
 type PayloadType = {
```

Comment:
It's a good idea to make this look like CSSOM properties for maintainers and users.

```suggestion
type Branding = Partial<{
  logoUrl: string,
  backgroundColor,
  color,
  fontFamily
  ...
```

Reply:
For example `color` instead of `fontColor` and `backgroundColor` instead of `color` unless `color` means something else for now.

Reply:
Can we change the API at the same time or do we have backwards compatibility concerns? Is the payload in question used by the web app and <Notification-Center/>?

Reply:
I imagined so.

Reply:
Noop for now :)